OneTimePassword/Token+URL.swift

```diff
@@ -78,7 +87,7 @@ private func algorithmFromString(string: String) -> Generator.Algorithm? {
     return nil
 }
 
-private func urlForToken(name name: String, issuer: String, factor: Generator.Factor, algorithm: Generator.Algorithm, digits: Int) -> NSURL? {
+private func urlForToken(name name: String, issuer: String, factor: Generator.Factor, algorithm: Generator.Algorithm, digits: Int) throws -> NSURL {
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 148 characters (line_length)